android/src/com/mapswithme/maps/MwmActivity.java

```diff
@@ -1970,9 +1933,44 @@ public void onShareLocationOptionSelected()
     shareMyLocation();
   }
 
-  public void onLayerItemClicked(@NonNull Mode mode)
+  @Override
+  public void onLayerItemClick(@NonNull Mode mode)
   {
     closeFloatingPanels();
-    mMapButtonsController.toggleMapLayer(mode);
+    if (mMapButtonsController != null)
```

Comment:
What is the situation when this variable is null?

Reply:
This variable is annotated with `@Nullable` so I am doing this check to remove the warning